### ISSUE TITLE
Mouse on screen still

### ISSUE THREAD

**Issue by Ultrunz** [xander1998/k9#9]
I saw faxes closed one about this, I have this issue also.

**Comment by xander1998** [comment]
I am not sure why....

**Comment by Ultrunz** [comment]
Yea. I haven't changed anything but the Keys. Added my steam hex and that's it

**Comment by xander1998** [comment]
I have seen issues with some resources. But not sure what those are...

**Comment by Ultrunz** [comment]
I think it might be something more than this script alone.

https://github.com/ESX-Org/esx_dmvschool/issues/14

The NUI does the same on this

**Comment by xander1998** [comment]
If you are not using ESX I get no issues with the K9 cursor.

**Comment by Ultrunz** [comment]
Ahh its an ESX compatibility maybe. No worries then, Thanks again

**Comment by xander1998** [comment]
I just think a popular resource used in ESX is fucked up and nobody has found it yet.

**Comment by Ultrunz** [comment]
Well with the dmv fucked. I think someone will be hunting it soon. I've kept the script on just not enabled. Looking forward to your newer iteration of it too.

**Comment by xander1998** [comment]
Yeah. Been having issues with synchronizing animations. I have thought about creating it in C# to better the performance and to go back to using NativeUI. We will see what I decide 🤷‍♂️
